### DISCUSSION TITLE
7.0.6 Release

### DISCUSSION THREAD

**Discussion by MikeNeilson** [opendcs/opendcs#203] [release-votes]
I will likely make the release this weekend anyways as it's the end of the month and I should be doing this poll in the middle of the month. But please record your vote here. If voting no please indicate what is blocking the release.

NOTE: regression and "things not working at all" are blockers. Missing features, etc are not and can get in the next release.

Currently we have a major regression that should've stopped the last release but is at least now fixed so we're getting one out no matter what. But I'd still like a record of process.

NOTE2: Anyone is welcome to vote, users with commit access are a binding vote. Other votes are not binding but will be listened to in case of bad, no-workaround regressions. (If you haven't open a ticket with more detail.)

**Comment by agilmore2** [comment]
I've got a small comp retry fix to PR tomorrow that I'd like to get in. I'm
in favor of releasing with that fix plus the XML fixes.

**Reply by MikeNeilson** [reply]
I'll be putting the release together Sunday afternoon or Monday morning. so there's still some time.
